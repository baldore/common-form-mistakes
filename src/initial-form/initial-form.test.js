test('should initial structure', () => {
  const form = createForm()

  document.body.appendChild(form.render())
  form.init()
})
