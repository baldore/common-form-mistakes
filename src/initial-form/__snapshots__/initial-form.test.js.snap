// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should initial structure 1`] = `
<div
  data-error=""
>
  There was an error.
</div>
`;
